src/components/PerformanceWidget/charts/PerformanceChart.tsx

```
@@ -313,13 +325,38 @@ function generateSampleData(currentBalances: Record<string, number>, dateRange?:
   });
 }
 
+// Add a debounce function at the top of the file (after the imports)
+const debounce = (func: Function, wait: number) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)